client/rest-high-level/src/main/java/org/opensearch/client/IndicesClient.java

```diff
@@ -1008,56 +954,6 @@ public Cancellable getAsync(GetIndexRequest getIndexRequest, RequestOptions opti
         );
     }
 
-    /**
-     * Retrieve information about one or more indexes
-     *
-     * @param getIndexRequest the request
-     * @param options the request options (e.g. headers), use {@link RequestOptions#DEFAULT} if nothing needs to be customized
-     * @return the response
-     * @throws IOException in case there is a problem sending the request or parsing back the response
-     * @deprecated This method uses an old request object which still refers to types, a deprecated feature. The method
-     * {@link #get(GetIndexRequest, RequestOptions)} should be used instead, which accepts a new request object.
-     */
-    @Deprecated
-    public org.opensearch.action.admin.indices.get.GetIndexResponse get(
-        org.opensearch.action.admin.indices.get.GetIndexRequest getIndexRequest,
-        RequestOptions options
-    ) throws IOException {
-        return restHighLevelClient.performRequestAndParseEntity(
-            getIndexRequest,
-            IndicesRequestConverters::getIndex,
-            options,
-            org.opensearch.action.admin.indices.get.GetIndexResponse::fromXContent,
-            emptySet()
-        );
-    }
-
-    /**
-     * Retrieve information about one or more indexes
-     *
-     * @param getIndexRequest the request
-     * @param options the request options (e.g. headers), use {@link RequestOptions#DEFAULT} if nothing needs to be customized
-     * @param listener the listener to be notified upon request completion
-     * @deprecated This method uses an old request object which still refers to types, a deprecated feature. The method
-     * {@link #getAsync(GetIndexRequest, RequestOptions, ActionListener)} should be used instead, which accepts a new request object.
-     * @return cancellable that may be used to cancel the request
-     */
-    @Deprecated
-    public Cancellable getAsync(
-        org.opensearch.action.admin.indices.get.GetIndexRequest getIndexRequest,
-        RequestOptions options,
-        ActionListener<org.opensearch.action.admin.indices.get.GetIndexResponse> listener
-    ) {
-        return restHighLevelClient.performRequestAsyncAndParseEntity(
-            getIndexRequest,
-            IndicesRequestConverters::getIndex,
-            options,
-            org.opensearch.action.admin.indices.get.GetIndexResponse::fromXContent,
-            listener,
-            emptySet()
-        );
-    }
-
     /**
```

Comment:
Curious if there will be a follow up PR for rest of the deprecated methods? like `forceMerge`

Reply:
Yes. I have one staged already